server/routes/basicDetails.ts

```diff
@@ -252,18 +245,27 @@ export default function basicDetailsRoutes(
     breachNoticeSaved: boolean,
     selectedAddressId: number,
   ): SelectItem[] {
-    return [
+    const returnAddressList: SelectItem[] = [
       {
         text: 'Please Select',
         value: '-1',
         selected: true,
       },
-      ...addresses.map(address => ({
-        text: formatAddressForSelectMenuDisplay(address),
-        value: `${address.id}`,
-        selected: breachNoticeSaved && address.id === selectedAddressId,
-      })),
     ]
+    if (addresses) {
+      const orderedAddressList: SelectItem[] = arrangeSelectItemListAlphabetically(
+        addresses.map(address => ({
+          text: formatAddressForSelectMenuDisplay(address),
+          value: `${address.id}`,
+          selected: breachNoticeSaved && address.id === selectedAddressId,
+        })),
+      )
+
+      // eslint-disable-next-line prefer-spread
+      returnAddressList.push.apply(returnAddressList, orderedAddressList)
```

Comment:
ESLint is suggesting you use the spread operator here (`...`).  This should give the same result:
```suggestion
      returnAddressList.push(...orderedAddressList)
```

ref: https://eslint.org/docs/latest/rules/prefer-spread